### PR TITLE
[T4A5][W10-B1] Kenneth Tan Xin You

### PR DESCRIPTION
**Pull request by kennethtxytqw**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/tagging/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.data.tagging;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/tagging/Tagging.java]: Header comment is required here.

File: src/seedu/addressbook/data/tagging/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		for(Tag tag: tags){
+			if(!person.getTags().contains(tag)){
+				Tagging tagging = new Tagging(true, person, tag);
+				taggings.add(tagging);
+			}
+		}
+		return taggings;
+	}
+	
+	public Tagging(){
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/tagging/Tagging.java]: This constructor is unnecessary here as it does not make much sense to instantiate a tagging object with no useful information at all. If this is not expected behavior, then try not to implement it.

**Comment by OuyangDanwen** [comment]
@kennethtxytqw It is good that you follow the diagram faithfully, i.e. keeping the tagging list in AddressBook rather in Tagging. Please read through the comments inserted in commits and close the PR after doing that.

**Comment by kennethtxytqw** [comment]
@OuyangDanwen Thank you! I have taken note of the comments.

**Pull request closed without merging**